### DISCUSSION TITLE
Wrench icon is not working

### DISCUSSION THREAD

**Discussion by Drenyth** [primefaces/community#1369] [PrimeIcons]
I'm working on a page which list some tools that can be useful for my enterprise and I wanted to put the wrench icon. But this icon is not working, I tried with other icons and it worked. So I'm wondering if I did something wrong or if it's a bug. If the bug is known, is there a patch soon ?

**Comment by Uzeyir9695** [comment]
same problem with me. After updating from 3.48.1 to 3.49.1 all icons disappeared.

**Comment by tugcekucukoglu** [comment]
Do you see a hydration error? Could you share your environment?
